lsp-mode.el

```diff
@@ -232,6 +232,21 @@ It contains all of the clients that are currently regitered.")
   :type 'boolean
   :group 'lsp-mode)
 
+(defcustom lsp-eldoc-enable-hover t
+  "If non-nil, eldoc will display hover info when it is present."
+  :type 'boolean
+  :group 'lsp-mode)
+
+(defcustom lsp-eldoc-enable-signature-help nil
+  "If non-nil, eldoc will display signature help when it is present."
+  :type 'boolean
+  :group 'lsp-mode)
+
+(defcustom lsp-eldoc-prefer-signature-help nil
```

Comment:
Similar comment here: once we have requested the signature why do we still show the hover considering that lsp-ui will display the hover too?

Reply:
lsp-ui-doc does not display signature information by default

Reply:
If you are on a parameter you won't see the signature but the parameter hover info.